### PR TITLE
Create react component for icon - Closes #6223

### LINKED ISSUE #6223: Create react component for icon
### Description

Create component to show an icon with properties
- name
- size

### Acceptance Criteria

- Use SCSS constant
- Add sample to the main page for testing

### Additional Information
Related Issues: #6216 
- https://fonts.google.com/icons

### PR DESCRIPTION
**Pull request by nazarhussain**
### What was the problem?

This PR resolves #6223

### How was it solved?

- Created `Icon` component 

### How was it tested?

- Manually added the component to page and tested with different sizes

### CODE REVIEW

File: framework-plugins/lisk-framework-dashboard-plugin/src/ui/app.tsx [hunk 1]
```diff
@@ -1,10 +1,12 @@
 import * as React from 'react';
 import { Provider } from 'react-redux';
 import { store } from './store';
-import { Main } from './pages/main';
+import MainPage from './pages/MainPage';
```
Comment (shuse2) [inline framework-plugins/lisk-framework-dashboard-plugin/src/ui/app.tsx:5]: this file needs to be renamed too?

File: framework-plugins/lisk-framework-dashboard-plugin/src/ui/components/Icon/index.tsx [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+interface Props {
+	name: string;
+	size?: 's' | 'm' | 'l' | 'xl';
+}
+
+const Icon: React.FC<Props> = props => {
+	const { name } = props;
+	const size = props.size ?? 'm';
+	return (
+		<span className={` ${styles.default.icon} ${styles.default[`icon-${size}`]}`}>{name}</span>
```
Comment (shuse2) [inline framework-plugins/lisk-framework-dashboard-plugin/src/ui/components/Icon/index.tsx:26]:
```suggestion
		<span className={`${styles.default.icon} ${styles.default[`icon-${size}`]}`}>{name}</span>
```

File: framework-plugins/lisk-framework-dashboard-plugin/src/ui/components/Icon/index.tsx [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+ *
+ * Unless otherwise agreed in a custom licensing agreement with the Lisk Foundation,
+ * no part of this software, including this file, may be copied, modified,
+ * propagated, or distributed except according to the terms contained in the
+ * LICENSE file.
+ *
+ * Removal or modification of this copyright notice is prohibited.
+ */
+import * as React from 'react';
+import * as styles from './Icon.module.scss';
```
Comment (shuse2) [inline framework-plugins/lisk-framework-dashboard-plugin/src/ui/components/Icon/index.tsx:15]:
```suggestion
import styles from './Icon.module.scss';
```

**Review by shuse2 — CHANGES_REQUESTED** [review]

File: framework-plugins/lisk-framework-dashboard-plugin/src/ui/app.tsx (hunk 1 above)
Comment (nazarhussain) [reply framework-plugins/lisk-framework-dashboard-plugin/src/ui/app.tsx:5]: It seems to be renamed. 
image

Comment (shuse2) [reply framework-plugins/lisk-framework-dashboard-plugin/src/ui/app.tsx:5]: ah sorry wrong line. `App.tsx` and `Text/index.tsx` is not changed

**Review by mitsuaki-u — APPROVED** [review]

**Review by shuse2 — APPROVED** [review]

**Pull request merged**